backend/src/main/java/moaon/backend/article/service/ESArticleService.java

```diff
@@ -32,6 +38,23 @@ public ArticleResponse search(ArticleESQuery condition) {
         return ArticleResponse.from(articles, totalHits, false, null);
     }
 
+    private ESArticleQueryBuilder getElasticsearchQueryBuilder(ArticleESQuery condition) {
+        ESArticleQueryBuilder builder = new ESArticleQueryBuilder();
+        builder.withTextSearch(condition.search())
+                .withSector(condition.sector())
+                .withTechStacksAndMatch(condition.techStackNames())
+                .withTopicsAndMatch(condition.topics())
+                .withSort(condition.sortBy());
+        ESCursor esCursor = condition.cursor();
+        if (esCursor.isEmpty()) {
+            builder.withPagination(0, condition.limit());
+        } else {
+            builder.withPagination(condition.limit(), esCursor.getSortValues());
```

Comment:
```suggestion
        if (esCursor.isEmpty()) {
            return builder.withPagination(0, condition.limit());
        }
        return builder.withPagination(condition.limit(), esCursor.getSortValues());
```

else 를 사용하지 않는 방향으로 제안 한 번 해봅니다 😄